util/session/sessionmanager.go

```diff
@@ -86,7 +92,12 @@ func (mgr *SessionManager) Create(subject string, secondsBeforeExpiry int) (stri
 		expires := now.Add(time.Duration(secondsBeforeExpiry) * time.Second)
 		claims.ExpiresAt = expires.Unix()
 	}
-	return mgr.signClaims(claims)
+
+	token, err := mgr.signClaims(claims)
+	if err != nil {
+		return nil, err
+	}
+	return &JwtToken{Token: token, IssuedAt: now.Unix()}, nil
```

Comment:
See previous comment. IssuedAt is already being generated in the claims so we don't need a wrapper around the token string.

Reply:
fixed